www/src/App.js

```diff
@@ -136,4 +141,23 @@ function SectionRoute() {
   return <Section spec={spec} section={section} />;
 }
 
+function ListRoute() {
+  const requirements = specifications.flatMap((spec) => spec.requirements);
+  const total = {
+    id: "AllSpecifications",
+    stats: specifications.reduce((total, {stats}) => {
+      Object.keys(stats).forEach((statName) => {
+        const stat = total[statName] || new StatsClass()
+        total[statName] = stat;
+        stat.onStat(stats[statName]);
+      });
+      return total;
+    }, {}),
+    requirements,
+  }
```

Comment:
Can you move this to `result.js` so that we're not recomputing stats every time you load this page? Or at least cache the value.

Reply:
Sure, it is not a lot of work and everything is local,
so I was not too worried, but :)